### DISCUSSION TITLE
Quarkus Rest Reactive Endpoints and Fault Tolerance

### DISCUSSION THREAD

**Discussion by jaiprasad266** [quarkusio/quarkus#26847] [Q&A] [answered]
Hi  ,

My question is on Quarkus Rest Reactive Endpoints and Fault Tolerance

I have an endpoint which connects to postgres db and returns the data back to the client.
And the endpoint looks like below

public Multi getData(RequestBody body) { //application code }

Below are the things I would like to understand :

What is the maximum concurrent http connections supported in quarkus rest reactive?
What happens if the influx of request exceeds the maximum allowed connections.

How backpressure is handled in Mutiny ? Do we need to handle it explicitly.

How to implement circuit breaker for Mutiny , where return type is say Multi

[a comment by an automated account is not shown]

**Reply by Ladicek** [reply]
This is a copy of https://stackoverflow.com/questions/73055415/quarkus-rest-reactive-endpoints-and-fault-tolerance. I personally consider it rather rude to post the same question on multiple places, especially if it's actually multiple questions and each of them is pretty open-ended.

**Reply by jaiprasad266** [reply]
in the past I had posted questions in this forum but I didnt get an replies , so I had posted this in both places . so I am really sorry , I will close this , right away

**Comment by jaiprasad266** [comment, marked as answer]
Go to this link for discussion on these questions.
https://stackoverflow.com/questions/73055415/quarkus-rest-reactive-endpoints-and-fault-tolerance.
